tests/src/tests/da/disperse.rs

```diff
@@ -22,13 +23,16 @@ async fn disseminate_and_retrieve() {
     let executor = &topology.executors()[0];
     let validator = &topology.validators()[0];
 
+    tokio::time::sleep(Duration::from_secs(15)).await;
```

Comment:
Why 15 seconds? Does setting up the test channel fail if we do it too quickly? Is there a ready check we can do instead of this 15s delay (i.e. maybe we wait until chain height grows to 1?) (feels like it could be a flaky test in the future)